### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by benfoley**
Lots of formatting changes to the code based on eslint config. No auto checking or pre-commit yet.

### CODE REVIEW

File: elpis/gui/src/components/App.js [hunk 1]
```diff
@@ -1,64 +1,54 @@
-import React, {Component} from 'react';
+import React, {Component} from "react";
```
Comment (mattchrlw) [inline elpis/gui/src/components/App.js:1]: In other code, I've seen spaces for import brackets & no spaces everywhere else, although I think that's personal preference as well :)

Comment (mattchrlw) [inline elpis/gui/src/components/App.js:1]: In other code, I've seen spaces for import brackets & no spaces everywhere else, although I think that's personal preference as well :)

**Review by mattchrlw — APPROVED** [review]
This looks good, I'm trusting there are no hidden functionality changes in these 69 diffs 😂

**Review by nicklambourne — CHANGES_REQUESTED** [review]

File: elpis/gui/.eslintrc [hunk 2]
```diff
@@ -7,6 +7,11 @@
     "eslint:recommended",
     "plugin:react/recommended"
   ],
+  settings: {
+    react: {
+      version: 'latest',
+    },
+  },
```
[the remark this answers is not in the document]
Comment (benfoley) [reply elpis/gui/.eslintrc:14]: there's a "detect" value which would save having to remember to update this whenever we change react version

**Pull request merged**